package.json

```diff
@@ -4,19 +4,24 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "test": "jest",
+    "emulator": "/home/cmenedes/Android/Sdk/tools/emulator -avd Nexus_5X_API_23 -netdelay none -netspeed full"
```

Comment:
This is specific to `cemenedes`. Is there a way we can make this work for all new developers?

Reply:
Just noticed this before. I think we can make it simple and just provide instructions for setting up and running an emulator. I can't run IOS, so this is Android specific. Instructions can contain setting up for both environments.  I'll remove this line of code. Does that sound good?

Reply:
That sounds great. Let's remove this line and really nail down the installation instructions.

Reply:
rm 'emulator' script